tools/cell_census_builder/src/cell_census_builder/build_soma/validate_soma.py

```diff
@@ -136,7 +137,8 @@ def validate_all_soma_objects_exist(soma_path: str, experiment_specifications: L
                 assert sum([c.non_zero_length for c in rna["feature_dataset_presence_matrix"].read().coos()]) > 0
                 # TODO(atolopko): validate 1) shape, 2) joinids exist in datsets and var
 
-        return True
+    gc.collect()
```

Comment:
Worth logging here as well?
```suggestion
    gc.collect()
    log_process_resource_status()
```

Reply:
added